scripts/postcss/css-url-processor.js

```diff
@@ -39,7 +39,7 @@ function colorsFromURL(url, colorMap) {
 function processURL(decl, replacer, colorMap) {
     const value = decl.value;
     const parsed = valueParser(value);
-    parsed.walk(async node => {
+    parsed.walk(node => {
```

Comment:
This change will disappear once https://github.com/vector-im/hydrogen-web/pull/721 merges